contrib/ecl/template.py

```diff
@@ -0,0 +1,59 @@
+pkgname = "ecl"
+pkgver = "23.9.9"
+pkgrel = 0
+build_style = "configure"
+configure_args = ["--prefix=/usr"]
+make_cmd = "gmake"
+hostmakedepends = ["gmake"]
+makedepends = [
+    "gc-devel",
+    "gmp-devel",
+    "libatomic_ops-devel",
+    "libffi-devel",
+]
+pkgdesc = "Embeddable Common Lisp"
+maintainer = "Paul A. Patience <paul@apatience.com>"
+license = "LGPL-2.1-or-later"
+url = "https://ecl.common-lisp.dev"
+source = f"https://gitlab.com/embeddable-common-lisp/{pkgname}/-/archive/{pkgver}/{pkgname}-{pkgver}.tar.gz"
+sha256 = "2ccda77461f651089d474f57d34c2fcee380ae54e734779b1bf4a4126e2a4686"
+
+
+def post_install(self):
+    self.install_license("COPYING")
+    self.install_license("LICENSE")
+
+    self.rm(self.destdir / f"usr/lib/ecl-{pkgver}/COPYING")
+    self.rm(self.destdir / f"usr/lib/ecl-{pkgver}/LICENSE")
+    self.rm(self.destdir / f"usr/lib/ecl-{pkgver}/build-stamp")
+
+
+@subpackage("ecl-static")
+def _static(self):
+    def func():
+        self.take(f"usr/lib/ecl-{pkgver}/*.a")
+
+    return func
```

Comment:
you can remove this as a ecl-devel-static is automatically created from the default_devel already

Reply:
(seems you'll need that `usr/lib/ecl-{pkgver}/*.a` in the extra= of devel)